indy_node/test/validator_info/test_validator_info.py

```diff
@@ -187,3 +186,12 @@ def test_validator_info_file_metrics_count_all_ledgers_field_valid(node):
     info = node._info_tool.info
     has_cnt = len(info['Node_info']['Metrics']['transaction-count'])
     assert has_cnt == len(new_ids) + 3
+
+
+def test_validator_info_update_date_field_valid(info):
+    assert "Update time" in info
+    import time
+    import datetime
+    from_str = time.mktime(datetime.datetime.strptime(info["Update time"],
+                                                      "%A, %B %d, %Y %I:%M:%S %p %z").timetuple())
+    assert int(from_str) == info["timestamp"]
```

Comment:
What is the difference between "Update Time" and "timestamp"?

Reply:
timestamp is an int value
Update Time is human readable string with tz offset